apps/site/src/app/(main)/apply/sections/Form/VolunteerForm.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+
+import { FormEvent, useState } from "react";
+import axios from "axios";
+
+import Button from "@/lib/components/Button/Button";
+
+import BasicInformation from "./BasicInformation";
+import AgeInformation from "./AgeInformation";
+import SchoolInformation from "./SchoolInformation";
+import ShiftAvailability from "./volunteer/ShiftAvailability";
+
+import styles from "./Form.module.scss";
+import hasDeadlinePassed from "@/lib/utils/hasDeadlinePassed";
+import VolunteerFRQ from "./volunteer/VolunteerFRQ";
+import ExtraQuestions from "./ExtraQuestions";
+
+const APPLY_PATH = "/api/user/apply";
+const FIELDS_WITH_OTHER = ["pronouns", "ethnicity", "school", "major"];
+
+export default function VolunteerForm() {
```

Comment:
Looks like all of the form logic is the same/very similar to the hacker application. Could we extract that into a reusable component that accepts the form fields as children?